2.TXT

```diff
@@ -0,0 +1,13 @@
+  INSERT INTO FOLKS (first_name, last_name, age)
+  VALUES ('Jey', 'Street', 36);
+ 
+  INSERT INTO FOLKS (last_name, first_name, age)
+  VALUES ('Silent','Bob', 41);
+   
+  INSERT INTO FOLKS (first_name, last_name)
+  VALUES ('Vadym','Marchenko');
+  
+  INSERT INTO FOLKS VALUES (4,'Doggo','Mexicanos', 3);
+  
+  INSERT INTO FOLKS
+  SELECT 5, 'Garold', 'Kumarenko', age FROM folks WHERE id = 1
```

Comment:
What is this select for?

Reply:
This select is part of insert statement. For example if we need to add some record to the table and one of columns must contain same information as we already have in the table. In this example - age.

Reply:
It looks strange, but okay